.jenkins

```diff
@@ -0,0 +1,114 @@
+pipeline {
+  agent none
+
+  options {
+    disableConcurrentBuilds(abortPrevious: true)
+    timeout(time: 6, unit: 'HOURS')
+  }
+
+  triggers {
+    issueCommentTrigger('.*test this please.*')
+  }
+
+  stages {
+    stage('Build') {
+      parallel {
+        stage('CUDA-12.6-NVCC') {
+          agent {
+              dockerfile {
+                  filename 'Dockerfile.nvcc'
+                  dir 'scripts'
+                  label 'nvidia-docker'
+                  args '--env NVIDIA_VISIBLE_DEVICES=$NVIDIA_VISIBLE_DEVICES'
+              }
+          }
+          steps {
+            sh '''rm -rf kokkos &&
+                  git clone https://github.com/kokkos/kokkos.git && \
+                  cd kokkos && \
+                  mkdir build && cd build && \
```

Comment:
I would prefer if we used modern cmake (cmake -B build; cmake--build build -j<N>; cmake --install build --prefix <prefix>)